### DISCUSSION TITLE
How does one specify templates?

### DISCUSSION THREAD

**Discussion by mil-ad** [datapane/datapane#351] [Help] [answered]
I've been staring at the API docs for a while now and I can't figure out how one specifies a template when creating an `App()` object!

**Comment by mil-ad** [comment]
The docs are sooo thin

**Comment by mil-ad** [comment]
I'm trying to `DescriptivePagesTemplate` but not sure where to import it from

**Comment by shahinrostami** [comment, marked as answer]
Hello @mil-ad.

The templates are a feature of the Jupyter integration, i.e. an `App` generated through `dp.App.from_notebook()`.

To set a template, you can use the `template` parameter, e.g. for your desired template above:

```python
dp.App.from_notebook(template="descriptive_pages")
```

If you run it without a template, we'll select one automatically - this includes a list of templates you can specify.

<img width="687" alt="Screenshot 2023-02-22 at 10 11 32" src="https://user-images.githubusercontent.com/15690380/220589600-53c58d0d-23da-4068-8661-5ee13dd81395.png">

I hope this helps! Please let us know if there's anything else.
